lib/rb_shift/deployment_config.rb

```diff
@@ -8,10 +8,12 @@ module RbShift
   # Representation of OpenShift deployment config
   class DeploymentConfig < OpenshiftKind
     def scale(replicas)
+      log.info "Scaling deployment config: #{name}"
       @parent.execute "scale dc #{name}", replicas: replicas
     end
 
     def start_deployment(block = false, timeout = 10)
+      log.info "Starting deployment config: #{name}"
```

Comment:
Nope, you're not starting deployment config, you're starting deploymetn from deployment config.